components/CompletionScoreCard.tsx

```diff
@@ -0,0 +1,213 @@
+'use client';
+
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Link from '@mui/material/Link';
+import FiberManualRecordIcon from '@mui/icons-material/FiberManualRecord';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import Chip from '@mui/material/Chip';
+
+interface CompletionScoreCardProps {}
+
+interface MockData {
+  title: string;
+  linkText: string;
+  linkHref: string;
+  scoreData: scoreData[];
+}
+
+interface scoreData {
+  status: 'High' | 'Medium' | 'Low';
+  total: number;
+  completed: number;
+}
+
+const mockData = {
+  title: 'Zero COventry',
+  linkText: 'netzerocities.kausal.tech/coventry-netzero',
+  linkHref: '#',
+  scoreData: [
+    { status: 'High', total: 13, completed: 10 } as scoreData,
+    { status: 'Medium', total: 10, completed: 7 } as scoreData,
+    { status: 'Low', total: 8, completed: 2 } as scoreData,
+  ],
+};
+
+const calculateScores = (scoreData: scoreData[]) => {
+  return scoreData.map((data) => ({
+    status: data.status,
+    score: `${data.completed}/${data.total}`,
+  }));
+};
+
+const calculatePercentage = (scoreData: scoreData[]) => {
+  const totalCompleted = scoreData.reduce(
+    (acc, curr) => acc + curr.completed,
+    0
+  );
+  const totalItems = scoreData.reduce((acc, curr) => acc + curr.total, 0);
+  return totalItems === 0 ? 0 : (totalCompleted / totalItems) * 100;
+};
+
+export const CompletionScoreCard: React.FC<CompletionScoreCardProps> = () => {
+  const [data, setData] = useState<MockData | null>(null);
+  const [scores, setScores] = useState<
+    { status: 'High' | 'Medium' | 'Low'; score: string }[]
+  >([]);
+  const [completionPercentage, setCompletionPercentage] = useState<number>(0);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setData(mockData);
+      const calculatedScores = calculateScores(mockData.scoreData);
+      setScores(calculatedScores);
+      const percentage = calculatePercentage(mockData.scoreData);
+      setCompletionPercentage(percentage);
+    }, 1000);
+  }, []);
+
+if (!data) {
+  return <Typography>Loading...</Typography>;
+}
```

Comment:
I think there's some indentation issue here, I forgot to add prettier to the project so I'll set that up (no need to fix this)